apps/src/javalab/Javalab.js

```diff
@@ -18,7 +18,8 @@ import {showLevelBuilderSaveButton} from '@cdo/apps/code-studio/header';
 import {RESIZE_VISUALIZATION_EVENT} from '@cdo/apps/lib/ui/VisualizationResizeBar';
 import Neighborhood from './Neighborhood';
 import NeighborhoodVisualizationColumn from './NeighborhoodVisualizationColumn';
-import DefaultVisualization from './DefaultVisualization';
```

Comment:
should we delete the DefaultVisualization file now?

Reply:
Already done :) git tracked it as a rename to TheaterVisualization (bit odd... but I guess the files were similar enough)

Reply:
ah weird, I should have looked closer at the file names!